rcl/test/rcl/test_node.cpp

```diff
@@ -745,3 +745,48 @@ TEST_F(CLASSNAME(TestNodeFixture, RMW_IMPLEMENTATION), test_rcl_node_names) {
     EXPECT_EQ(RCL_RET_OK, ret);
   }
 }
+
```

Comment:
Looking at the coverage report, the `fail` label in `rcl_node_init` is never called. Is a test that exercises that part of the file easily addable?

Reply:
I didn't see any easily addable. I was covering missing API tests with this PR, and probably I'll get to that part once I make another round adding "bomb alllocators".